demo/react-hooks-controlled-multiple.html

```diff
@@ -49,64 +50,56 @@ <h2>Controlled stack</h2>
         // no need to init twice (would will return same grid) or register dup events
         const grid = gridRef.current = GridStack.init(
           {
-            float: true,
+            float: false,
             acceptWidgets: true,
             disableOneColumnMode: true, // side-by-side and fever columns to fit smaller screens
             column: 6,
             minRow: 1,
           },
-          `.controlled-${id}`
+          gridContainerRef.current
         )
-        .on('added', (ev, gsItems) => {
-          if (grid._ignoreCB) return;
-          // remove the new element as React will re-create it again (dup) once we add to the list or we get 2 of them with same ids but different DOM el!
-          // TODO: this is really not ideal - we shouldn't mix React templating with GS making it own edits as those get out of sync! see comment below @111.
-          gsItems.forEach(n => {
-            grid.removeWidget(n.el, true, false); // true=remove DOM, false=don't call use back!
-            // can't pass n directly even though similar structs as it has n.el.gridstackNode which gives JSON error for circular write.
-            addItem({id:n.id, x:n.x, y:n.y, w:n.w, h:n.h});
-          });
-        })
-        .on('removed', (ev, gsItems) => {
-          if (grid._ignoreCB) return;
-          gsItems.forEach(n => removeItem(n.id));
-        })
-
+          .on('added', (ev, gsItems) => {
+            if (grid._ignoreCB) return;
+            // remove the new element as React will re-create it again (dup) once we add to the list or we get 2 of them with same ids but different DOM el!
+            // TODO: this is really not ideal - we shouldn't mix React templating with GS making it own edits as those get out of sync! see comment below @111.
+            gsItems.forEach(n => {
+              grid.removeWidget(n.el, true, false); // true=remove DOM, false=don't call use back!
+              // can't pass n directly even though similar structs as it has n.el.gridstackNode which gives JSON error for circular write.
+              addItem && addItem({ id: n.id, x: n.x, y: n.y, w: n.w, h: n.h });
```

Comment:
so only read change is addItem && - why would it ever be null ?

Reply:
It was done to avoid errors in the case if `addItem === undefined` i.e. wasn't provided, but ok i have reverted it back